lib/sentry_logging.rb

```diff
@@ -31,13 +29,18 @@ def log_exception_to_sentry(exception, extra_context = {}, tags_context = {}, le
     end
     rails_logger(level, exception.backtrace.join("\n")) unless exception.backtrace.nil?
   end
-  # rubocop:enable Metrics/CyclomaticComplexity
-  # rubocop:enable Metrics/PerceivedComplexity
 
-  def normalize_level(level)
+  def normalize_level(level, exception = nil)
     # https://docs.sentry.io/clients/ruby/usage/
     # valid raven levels: debug, info, warning, error, fatal
-    level = level.to_s
+    level = if exception.is_a?(Pundit::NotAuthorizedError)
```

Comment:
case statements can match on classes and i feel like they are easier to read and change:
```
level = case exception
            when Pundit::NotAuthorizedError
              'info'
            when Common::Exceptions::BaseError
              exception.sentry_type.to_s
            else
              level
            end
```